apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -180,8 +181,18 @@ class TopInstructions extends Component {
     hidden: PropTypes.bool.isRequired,
     shortInstructions: PropTypes.string,
     isMinecraft: PropTypes.bool.isRequired,
+    isBlockly: PropTypes.bool.isRequired,
     isRtl: PropTypes.bool.isRequired,
-    widgetMode: PropTypes.bool
+    widgetMode: PropTypes.bool,
+    isCSF: PropTypes.bool,
```

Comment:
How is isCSF different than collapsible? My understanding is they have meant the same things in instructions

Reply:
`noInstructionsWhenCollapsed` means the same thing as `!isCSF` currently so there is some redundancy. I can remove `isCSF` and just use `noInstructionsWhenCollapsed`, I just felt like that made the code in LevelDetailsDialog more confusing. But I think it makes this code a bit more confusing...

Reply:
Ya I get that its confusing. I just wonder if the consistency is helpful and that maybe we should be working on changing the variable entirely to better reflect what it does across the board?

Reply:
Which would be outside the scope of this

Reply:
Yeah, I'm leaning towards what you're saying about keeping consistency. I removed the `isCSF` and `collapsible` props from TopInstructions. Some of the downstream components still use `collapsible` because that still seems more descriptive but I can change those back too if you think it'll be clearer

Reply:
I think thats fine. Thanks for updating.